src/Common/src/Interop/Unix/System.Native/Interop.Permissions.cs

```diff
@@ -5,7 +5,7 @@
 
 internal static partial class Interop
 {
-    internal static partial class libc
+    internal static partial class Sys
     {
         [Flags]
         internal enum Permissions
```

Comment:
I considered making them hex on the native side for the sake of consistency, but octal and permissions go together like peanut butter and jelly. :)